src/main/java/com/google/api/codegen/viewmodel/metadata/PackageMetadataView.java

```diff
@@ -90,7 +98,7 @@ public String resourceRoot() {
   public abstract List<String> typeModules();
 
   public static Builder newBuilder() {
-    return new AutoValue_PackageMetadataView.Builder();
+    return new AutoValue_PackageMetadataView.Builder().hasSmokeTests(false);
```

Comment:
This is necessary. The default behavior is to throw an error if the field is not set.